plugin/src/main/java/io/jenkins/plugins/casc/Attribute.java

```diff
@@ -275,7 +323,12 @@ public CNode describe(Owner instance, ConfigurationContext context) throws Confi
      */
     private CNode _describe(Configurator c, ConfigurationContext context, Object value, boolean shouldBeMasked)
             throws Exception {
-        CNode node = c.describe(value, context);
+        CNode node;
+        if(isJsonSchema) {
```

Comment:
@casz Added this so that it would not throw any casting errors, since in the descirbe structure we were initally calling with value whereas now we needed the type. This reduces the castClassException

Reply:
is this still required?